lib/node_modules/@stdlib/stats/base/dsmeanors/include/stdlib/stats/base/dsmeanors.h

```diff
@@ -29,9 +29,14 @@ extern "C" {
 #endif
 
 /**
-* Computes the arithmetic mean of a single-precision floating-point strided array using ordinary recursive summation with extended accumulation and returning an extended precision result.
+* Computes the arithmetic mean of a single-precision floating-point strided array `x` using ordinary recursive summation with extended accumulation and returning an extended precision result.
```

Comment:
```suggestion
* Computes the arithmetic mean of a single-precision floating-point strided array using ordinary recursive summation with extended accumulation and returning an extended precision result.
```

Reply:
If you are going to include a variable name, it better match the signature. Here, that is `X`, not `x`. This sort of typo is one of the reasons we have been updating docs to simply refer to the "input strided array", rather than the explicit `x` or `y`.